contracts/src/v0.8/automation/interfaces/zksync/IZKSyncAutomationRegistryMaster2_3.sol

```diff
@@ -2,7 +2,7 @@
 // abi-checksum: 0x5857a77a981fcb60dbdac0700e68420cbe544249b20d9326d51c5ef8584c5dd7
 // SPDX-License-Identifier: MIT
 // !! THIS FILE WAS AUTOGENERATED BY abi-to-sol v0.6.6. SEE SOURCE BELOW. !!
-pragma solidity ^0.8.4;
+pragma solidity ^0.8.19;
```

Comment:
was this change auto-gened or can this get overwritten when someone regenerates the file next time?